src/components/save.tsx

```diff
@@ -0,0 +1,128 @@
+import * as React from "react"
```

Comment:
Since the component will only be used inside of the artwork grid item component, could you move it to the `artwork` folder?